openedx/core/djangoapps/content/block_structure/transformer_registry.py

```diff
@@ -50,7 +50,7 @@ def get_write_version_hash(cls):
         sorted_transformers = sorted(cls.get_registered_transformers(), key=lambda t: t.name())
         for transformer in sorted_transformers:
             hash_obj.update(transformer.name().encode('utf-8'))
-            hash_obj.update(str(transformer.WRITE_VERSION))
```

Comment:
Do you need to seave the call to `str` still?  That seems redundant.

Reply:
yeah. without it it gives error.